src/cryptography/x509/extensions.py

```diff
@@ -865,6 +865,9 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(OCSPNoCheck)
 
+    def __ne__(self, other):
```

Comment:
Ooops, one more nit, can you please move `__ne__` to be defined directly under `__eq__`?

Reply:
As you wish, of course! Fixed in bc15f6dc.

I generally try to sort functions alphabetically (that's also what Django prefers), so that's what I initially used. I want to note that e.g. `TLSFeature` also has `__ne__` separated from `__eq__`.